pages/about.html

```diff
@@ -0,0 +1,122 @@
+---
+excerpt: Who developed Data.gov? With what technology is Data.gov built?
+link: https://www.data.gov/about
+redirect_from:
+- /faq/
+- /climate/faq/
+slug: about
+title: About Us
+layout: layouts/page-wide
+---
+
+<div id="top"></div>
+
+<div class="grid-container">
+  <div class="usa-prose">
+    <h1 id="about">{{ title }}</h1>
+  </div>
+  <section class="usa-section">
+    <div class="grid-row grid-gap-lg">
+      <div class="grid-col-12 desktop:grid-col-5 ">
+        <h4 id="mission">Mission</h4>
+        <div>
+          The United States Government's open data site is designed to unleash the power of government open data to
+          <b>inform decisions by the public and policymakers, drive innovation and economic activity, achieve agency
```

Comment:
```suggestion
          <b>inform decisions of the public and policymakers, drive innovation and economic activity, achieve agency
```
I would suggest either "of the public" or "made by the public".  "by the public" seems a bit too ambiguous.

Reply:
Clear enough as is.